targets/csharp/source/Plugins/CloudScript/source/PlayFabFunctionContexts.cs

```diff
@@ -1,4 +1,9 @@
-﻿#if NETSTANDARD2_0
+/////////////////////////////////////////////////////////////////////////////////////////////////
+/// THIS PLUGIN IS CURRENTLY IN A BETA STATE MEANT TO ONLY BE USED BY PRIVATE PREVIEW CUSTOMERS.
```

Comment:
This sounds like this file would be a good candidate to be in a separate beta repo instead of our public repo. The comment is fairly easy to understand, but it seems like anything we put in the public repo should be expected to be scrutinized and potentially used.

Reply:
The thing with this file is that it's only really useful to create an object for you out of a payload sent to your Azure Function. Non private-preview customers do not have anything to do with this file for now cause they won't have the required input coming in to this function, even if they try to use it.

Reply:
As long as you're fine with the fact that some customers WILL find the file, and the comments, I don't strongly oppose its existence.
We will have the ability to keep features in private repos soon, so we can revisit this after that.